includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -210,8 +210,11 @@ protected function get_phone_number_args( $order ) {
 				'night_phone_c' => substr( $phone_number, 6, 4 ),
 			);
 		} else {
+			$calling_code = WC()->countries->get_calling_code_for_country( $order->get_billing_country() );
+			$phone_number = $calling_code ? preg_replace( '/^0/', '', $order->get_billing_phone() ) : $order->get_billing_phone();
```

Comment:
We are removing `0` from the beginning, but is fine use `+`? Maybe we should remove too, or start including, since that list doesn't include for all numbers.
Also what happens if a customer already includes calling code? Probably we should remove it before including into `night_phone_b`.

Reply:
This won't work for some countries, as per [popular phone numbers resource](https://github.com/googlei18n/libphonenumber/blob/master/FALSEHOODS.md) bullet point 12 and 20. However, I guess it should work for most.

Reply:
@peterfabian yep, but probably depends on how PayPal wants that data, with that `+` or not...